site/pages/dashboard.page.tsx

```diff
@@ -84,26 +131,41 @@ const Dashboard = ({ liveDisruptions, upcomingDisruptions }: DashboardProps): Re
                     {
                         tabHeader: "Live",
                         content: (
-                            <Table
-                                caption="Live disruptions"
-                                columns={["ID", "Summary", "Affected dates"]}
-                                rows={formatDisruptionsIntoRows(liveDisruptions)}
-                            />
+                            <>
+                                <Table
+                                    caption="Live disruptions"
+                                    columns={["ID", "Summary", "Affected dates"]}
+                                    rows={formatDisruptionsIntoRows(liveDisruptionsToDisplay)}
+                                />
+                                <PageNumbers
```

Comment:
The gov guidance is to only show pagination if the number of pages is greater than 1 so we could put this behind a check on `numberOfLiveDisruptionsPages`